src/app/(proper_react)/redesign/(public)/LandingView.tsx

```diff
@@ -42,37 +43,79 @@ export const View = (props: Props) => {
             isHero
             eligibleForPremium={props.eligibleForPremium}
             signUpCallbackUrl={`${process.env.SERVER_URL}/redesign/user/dashboard/`}
+            eventId={{
+              cta: "clicked_get_scan_header",
+              field: "entered_email_address_header",
+            }}
           />
         </div>
         <div className={styles.heroImage}>
           <HeroImage {...props} />
         </div>
       </header>
 
-      <div className={styles.valuePropositionWrapper}>
+      <section className={styles.quoteWrapper}>
```

Comment:
`<main>` already serves as a section, so I think the `<section>`s in this file should just be `<div>`s.